src/Makefile

```diff
@@ -23,7 +23,10 @@ NETWORK_URI_DIR=$(CPP_NETLIB_DIR)/deps/uri
 NETWORK_URI_LIBDIR=$(NETWORK_URI_DIR)/src
 YAML_CPP_DIR=$(LIBDIR)/yaml-cpp
 YAML_CPP_LIBDIR=$(YAML_CPP_DIR)
-SUBMODULE_DIRS=$(CPP_NETLIB_DIR) $(YAML_CPP_DIR)
+PROMETHEUS_CPP_DIR=$(LIBDIR)/prometheus-cpp
+PROMETHEUS_CPP_LIBDIR=$(PROMETHEUS_CPP_DIR)
+PROMETHEUS_CPP_CORE_LIBDIR=$(PROMETHEUS_CPP_LIBDIR)/core
```

Comment:
I still have this variable as `PROMETHEUS_CPP_CORE_*` because it makes it clear for the reader that we only uses the `prometheus_cpp::core` library, not the `prometheus_cpp::pull` or `prometheus_cpp:push`.